src/main/java/ru/svgogin/service/spark/errordto/CompanyErrorDto.java

```diff
@@ -0,0 +1,31 @@
+package ru.svgogin.service.spark.errordto;
+
+import org.springframework.lang.NonNull;
+
+public final class CompanyErrorDto {
+  @NonNull
+  private final ErrorCode code;
+  @NonNull
+  private final String message;
+
+  public CompanyErrorDto(@NonNull ErrorCode code, @NonNull String message) {
```

Comment:
This annotation doesn't work here
```suggestion
  public CompanyErrorDto(ErrorCode code, String message) {
```

Reply:
Done